### PR TITLE
Montréal-Python 68: Adds vidéos of the talks

### PR DESCRIPTION
**Pull request by mlhamel**
Adds the videos of the talks for our latest event mp68 http://montrealpython.org/en/2017/11/mp68/

### CODE REVIEW

File: montreal-python/videos/mp68-va-debugger-ton-python-stephane-wirtel.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "copyright_text": "Creative Commons Attribution license (reuse allowed)",
+  "description": "Cette pr\u00e9sentation vous explique les bases de Pdb ainsi que GDB, afin de debugger plus facilement vos scripts Python.\n\nMontr\u00e9al-Python 68: Wysiwyg Xylophone \nhttps://montrealpython.org/2017/11/mp68/",
+  "recorded": "2017-11-20",
+  "language": "eng",
```
Comment (pior) [inline montreal-python/videos/mp68-va-debugger-ton-python-stephane-wirtel.json]: `"fre"` maybe?

**Comment by Daniel-at-github** [comment]
'fre' it's the logical option, but it should be 'fra' to be showed as 'French'.

The code that formats it: https://github.com/pyvideo/pyvideo/blob/master/pelicanconf.py#L100
~~~ py
# https://en.wikipedia.org/wiki/ISO_639-3
# https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes
VIDEO_LANGUAGE_NAMES = {
    'ita': 'Italian',
    'zho': 'Chinese',
    'por': 'Portuguese',
    'ukr': 'Ukrainian',
    'deu': 'German',
    'eng': 'English',
    'rus': 'Russian',
    'fra': 'French',
    'spa': 'Spanish',
    'eus': 'Basque',
    'cat': 'Catalan',
    'glg': 'Galician',
    'kor': 'Korean',
    'lit': 'Lithuanian',
    'jpn': 'Japanese',
    'slk': 'Slovak',
}
~~~

The documentation is on [CONTRIBUTING.rst](https://github.com/pyvideo/data/blob/master/CONTRIBUTING.rst#contributing)  
> language | string | (ISO 639-3 coded)

Feel free to improve documentation if it's confusing.

**Comment by mlhamel** [comment]
@redapple I've renamed `fre` to `fra` + removed the speaker name from the file name

**Comment by redapple** [comment]
Oh, I meant removing the speaker names from the `"title"` fields in the JSON data

**Comment by mlhamel** [comment]
Oups just fixed this, thanks for the heads up @redapple

**Pull request merged**